cmd/utils/flags.go

```diff
@@ -110,6 +110,30 @@ func NewApp(gitCommit, usage string) *cli.App {
 // are the same for all commands.
 
 var (
+	// Node Finder settings
+	MaxNoFileFlag = cli.Uint64Flag{
+		Name:  "maxnofile",
+		Usage: "Maximum file descriptor allowance of this process (try 1048576)",
```

Comment:
what is the significance of 1048576?

Reply:
that is what kingslanding/highgarden had as the hard limit.

Reply:
ack